splunk_eventgen/__main__.py

```diff
@@ -308,6 +308,13 @@ def build_splunk_app(dest, source=os.getcwd(), remove=True):
     directory_default_dir = os.path.join(directory, 'default', 'eventgen.conf')
     eventgen_conf = os.path.join(module_path, 'default', 'eventgen.conf')
     shutil.copyfile(eventgen_conf, directory_default_dir)
+
```

Comment:
Add `ujson` and `jinja2` dependency when building the app.

Reply:
Are we sure we want to agree for the install of these libraries for the end user?

Reply:
Both are BSD license based and I will check it with legal.

Reply:
Any update here?

Reply:
The process with OSS will start with all the 3rd party libs used by Eventgen.  Not just these two in app.